src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -8825,8 +8825,6 @@ void StorageReplicatedMergeTree::lockSharedData(
     auto zookeeper = tryGetZooKeeper();
     if (zookeeper)
         return lockSharedData(part, std::make_shared<ZooKeeperWithFaultInjection>(zookeeper), replace_existing_lock, hardlinked_files);
```

Comment:
It looks like `return` is also unneeded.

Reply:
I've fixed it.

Reply:
Thanks!